config/initializers/job_configurations.rb

```diff
@@ -1,4 +1,5 @@
 cron_5m = '0/5 * * * *'
+cron_12m = '0/5 * * * *'
```

Comment:
I don't have strong opinions on 12m vs 5m intervals, but with the `0/5` this is still saying "every 5 minutes"

I think we need:
```suggestion
cron_12m = '0/12 * * * *'
```

I used [crontab guru](https://crontab.guru/#0/12_*_*_*_*) to spot check these

Reply:
Oh dear. I updated that part only in my imagination, it seems. Thank you for catching this!